### ISSUE TITLE
[JRuby] msgpack 1.2 does not load

### ISSUE THREAD

**Issue by petr666** [msgpack/msgpack-ruby#148]
Hi,
"require msgpack" on JRuby fails with "no such file to load -- msgpack/msgpack". It seems there is no msgpack.jar bundled in gem.

**Comment by tagomoris** [comment]
Thank you for reporting it!
I re-released v1.2.1-java, and verified it on my laptop.
```
$ gem i msgpack
Fetching: msgpack-1.2.1-java.gem (100%)
Successfully installed msgpack-1.2.1-java
1 gem installed
$ jruby -rmsgpack -e 'p 1'
1
```

@petr666 Could you check it on your side too?

**Comment by petr666** [comment]
With 1.2.1 everything is working. Thanks for quick fix.
